tests/integration/telemetry/api/accesslogs_test.go

```diff
@@ -52,14 +55,35 @@ func TestAccessLogs(t *testing.T) {
 		Run(func(t framework.TestContext) {
 			t.NewSubTest("enabled").Run(func(t framework.TestContext) {
 				applyTelemetryResource(t, true)
-				runAccessLogsTests(t, true)
+				runAccessLogsTests(t, true, false)
+				deleteTelemetryResource(t, true)
+			})
+			t.NewSubTest("enabled-with-targetref").Run(func(t framework.TestContext) {
```

Comment:
Remember, Gateway API installs as CRDs, so you need this line to start off your test case: `crd.DeployGatewayAPIOrSkip(t)`